dotnet-directory

```diff
@@ -49,14 +49,14 @@ print_sdk() {
 }
 
 print_framework() {
-    if [[ $# -eq 0 ]]; then
+    if [[ "$2" -eq 0 ]]; then
         print_err "Missing <version> argument."
         exit 1
     fi
-    local version="$1"
+    local version="$2"
```

Comment:
Can you use a local for each argument? It serves as a form of documentation for the function arguments.

`local dir_name="$1"`